source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -79,10 +79,13 @@
                             window.location = urlGenerator.action(calificationPeriod, "Evaluations");
                         }
                     }
-                },
-                error: function () {
-                    alert('Fallo interno. Por favor recargue la página.');
                 }
+            })
+            .fail(function () {
+                alert('Fallo interno. Por favor recargue la página.');
```

Comment:
@andresmoschini I couldn't reproduce the expired session locally, I will merge and deploy to dev to reproduce it there, sound good to you??